src/components/Hero/HeroSection.astro

```diff
@@ -0,0 +1,107 @@
+---
+import CountUp from "react-countup";
+import GridWrapper from "./GridWrapper";
+import HeroSwiper from "./HeroSwiper";
+---
+
+<>
+  {/* Text of the HeroSection */}
+  <div class="flex max-md:grid">
+    <div class="p-10 w-[50%] max-md:w-[100%]">
+      <button class="flex bg-[#F2F4F7] p-[.5rem] rounded-[19px]">
+        <div class="px-3">Visit Our Guidelines</div>
+        {/* Svg for arrow in button */}
+        <svg
+          width="24"
+          height="24"
+          viewBox="0 0 24 24"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <rect width="24" height="24" rx="12" fill="#E4E7EC"></rect>
+          <path
+            d="M6.16663 12H17.8333M17.8333 12L12 6.16669M17.8333 12L12 17.8334"
+            stroke="#101828"
+            stroke-width="1.5"
+            stroke-linecap="round"
+            stroke-linejoin="round"></path>
+        </svg>
+      </button>
+      <div class="bold font-title text-[2.4rem] max-lg:text-[2rem]">
+        "Unlock Your Career Potential with Training And Placement Cell - Where
+        Dreams Meet Opportunities!"
+      </div>
+    </div>
+    <div class="w-[50%] p-10 justify-between flex flex-col max-md:w-[100%]">
+      <div
+        class="text-[#475467] text-[18px] max-lg:text-[1rem] leading-[28px] tracking-wider"
+      >
+        The Training and Placement department serves as a bridge between the
+        academic environment and the industry by establishing connections with
+        various companies and organizations. The ultimate aim of the T&P
+        department is to ensure that students are well-prepared for the job
+        market and have access to the resources they need to achieve their
+        career goals.
+      </div>
+      <button
+        class="my-6 p-3 rounded-2xl m-auto"
+        style={{ border: "2px solid #E5E5E5" }}
+      >
+        View More
+      </button>
+    </div>
+  </div>
+
+  {/* Numbers and text inside it */}
+  <div class="flex justify-evenly max-lg:grid">
+    <div class="ml-6 max-w-[40%] max-lg:max-w-[100%]">
+      <GridWrapper client:load columns={2}>
+        <div
+          class="max-w-[180px] p-4 m-3 rounded-2xl"
```

Comment:
don't use pixels. use rems. or default tailwind values max-w-16 something will be around 180px. hover over that utility in vscode to know the exact value

Reply:
done